cypress/e2e/court-cases-index/pagination.cy.ts

```diff
@@ -22,6 +22,7 @@ describe("Pagination", () => {
     cy.get("h1")
 
     cy.checkA11y(undefined, a11yConfig, logAccessibilityViolations)
+    console.log(logAccessibilityViolations)
```

Comment:
does this need to be left in? the cypress GUI will show a list of accessibility violations already

Reply:
I forgot to remove it. thanks for spotting